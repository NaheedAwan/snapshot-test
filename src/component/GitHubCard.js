import React from 'react'
import { Card } from 'react-bootstrap'
export default function GitHubCard() {
    return (
        <div>
            <Card style={{ width: '18rem' }}>
                <Card.Body>
                    <Card.Title>GitHub</Card.Title>
                    <Card.Subtitle className="mb-2 text-muted">NaheedAwan</Card.Subtitle>
                    <Card.Text>
                      My name is Naheed Awan, currently studying full stack web app development.
                      Student NorthCarolina State Universtiy, Raleigh-Durham.
                    </Card.Text>
                    <Card.Link href="#">Card Link</Card.Link>

                </Card.Body>
            </Card>
        </div>
    )
}
