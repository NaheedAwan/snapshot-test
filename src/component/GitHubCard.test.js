import GitHubCard from './GitHubCard'
import renderer from 'react-test-renderer'

// test('renders a snapshot', () => {
//   const tree = renderer.create(<GitHubCard />).toJSON()
//   expect(tree).toMatchSnapshot()
// })

test('renders text GitHub', () => {
    render(<App />)
    const nameElement = screen.getByText(/github/i)
    expect(nameElement).toBeInTheDocument()
  });
  