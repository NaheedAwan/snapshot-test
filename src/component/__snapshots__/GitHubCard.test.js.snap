// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div>
  <div
    className="card"
    style={
      Object {
        "width": "18rem",
      }
    }
  >
    <div
      className="css-1abc card-body"
    >
      <div
        className="card-title h5"
      >
        GitHub
      </div>
      <div
        className="mb-2 text-muted card-subtitle h6"
      >
        NaheedAwan
      </div>
      <p
        className="card-text"
      >
        My name is Naheed Awan, currently studying full stack web app development. Student NorthCarolina State Universtiy, Raleigh-Durham.
      </p>
      <a
        className="card-link"
        href="#"
      >
        Card Link
      </a>
    </div>
  </div>
</div>
`;
